app/main.py

```diff
@@ -1,7 +1,30 @@
-def get_weather() -> None:
-    # write your code here
-    pass
+import requests
+import os
+
+BASE_URL = "http://api.weatherapi.com/v1/current.json"
+
+
+def get_weather(api_key: str, CITY) -> None:
```

Comment:
Checklist point 1: Use constants. In your function definition, you have used `CITY` as a parameter. It's better to define it as a constant at the top of your file, just like you did with `BASE_URL`.